lldb/examples/python/crashlog.py

```diff
@@ -1794,8 +1803,36 @@ def SymbolicateCrashLogs(debugger, command_args, result, is_command):
         result.SetError(str(e))
         return
 
+    # To avoid breaking existing users, we should keep supporting legacy flags
+    # even if we don't use them / advertise them anymore.
+    if options.interactive:
+        options.mode = CrashLogLoadingMode.interactive
+    elif options.batch:
+        options.mode = CrashLogLoadingMode.batch
+
+    if (
+        options.mode
```

Comment:
Isn't `options.mode` always set to interactive? It defaults to that up in the argparse code, so is this check necessary?

Reply:
Oversight.